src/jaxsim/api/ode_data.py

```diff
@@ -158,6 +162,15 @@ def build_from_jaxsim_model(
             `JaxSimModel` and initialized to zero.
         """
 
+        # Get the contact model from the `JaxSimModel`
+        prefix = type(model.contact_model).__name__.split("Contact")[0]
+
+        if prefix:
+            module_name = f"{prefix.lower()}_contacts"
+            class_name = f"{prefix.capitalize()}ContactsState"
+        else:
+            raise ValueError("Unable to determine contact state class prefix.")
```

Comment:
Would `isinstance` on `model.contact_model` work here? You can import all supported contact model inside this method. In the future, we can think of better ways using multiple dispatch for allowing users to define custom models, if needed. However, I'd keep things simple for now.

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/9675c4965004291dd3d78cae1546ff539b9e3f80